package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@crowdstrike/foundry-js",
-  "version": "0.15.0",
+  "version": "0.16.0",
   "description": "foundry.js is the JavaScript SDK for authoring UI Extensions for CrowdStrike's Foundry platform.",
```

Comment:
Should this be foundry-js instead of foundry.js? There's a similar misspelling in the README. 

> `foundry.js` javascript library provides convenient access to the CrowdStrike's Foundry API to author UI pages and extensions.

This should probably be:

> The `foundry-js` JavaScript library provides convenient access to the CrowdStrike's Foundry API to author UI pages and extensions.

Reply:
https://github.com/CrowdStrike/foundry-js/pull/72